### ISSUE TITLE
[Bug]: Conflicting with ProductiveTrees(?) Crashing when added to verion 1.7.0 of Direworld20 1.21

### ISSUE THREAD

**Issue by ItMeCorban** [TeamLapen/Vampirism#1460]
### Minecraft Version

1.21.x

### (Neo)Forge Version

21.1.83

### Vampirism Version

1.10.0

### Modpack

FTB Direworld20 1.21 v1.7.0

### Singleplayer/Multiplayer

Server

### What happened?

I have a server setup for Direwolf20 1.21. The client is working fine when adding the mods, but the server is failing to load. Version 1.4.1 works as expected, so it's something in 1.7.0, specifically to the server. I can load a singleplayer world and everything works fine.

In the logs I see something fail with the datapack for productivetrees, but I'm not certain if that's related

### Relevant log output

https://pste.ch/erolusuyuv.md

### Other relevant mods

Productive Trees 1.21.1-0.6.1 (productivetrees)

**Comment by Cheaterpaul** [comment]
Yes, this seems to happen because ProductiveTrees generates its data resources on runtime without properly providing the correct registryobjects https://github.com/JDKDigital/productivetrees/issues/26

**Comment by ItMeCorban** [comment]
Ah okay so the issue is on productivetrees side, thanks for the insight, I'll follow the progress other there :)

**Comment by Cheaterpaul** [comment]
Seems they do not care for mc 1.21.
